terraform/main.tf

```diff
@@ -193,7 +146,6 @@ data "template_file" "ci_uploader" {
   vars = {
     bucket_name  = aws_s3_bucket.idp_id_broker_search.bucket
     bucket2_name = aws_s3_bucket.idp_id_broker_search_2.bucket
-    bucket3_name = aws_s3_bucket.idp_id_broker_search_3.bucket
```

Comment:
Based on the plan error, it looks like you may also need to remove the `bucket3_name` variable from that "ci-bucket-policy.json" file:

> Error: failed to render : <template_file>:14,25-37: Unknown variable; There is no variable named "bucket3_name". Did you mean "bucket2_name"?
with data.template_file.ci_uploader
on main.tf line 143, in data "template_file" "ci_uploader":
data "template_file" "ci_uploader" {

Reply:
Thanks